javalib/src/main/scala/java/util/concurrent/Flow.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.util.concurrent
+
+object Flow {
+
+  trait Processor[T, R] extends Subscriber[T] with Publisher[R]
+
+  trait Publisher[T] {
+    def subscribe(subscriber: Subscriber[T]): Unit
+  }
+
+  trait Subscriber[T] {
+    def onComplete(): Unit
+    def onError(throwable: Throwable): Unit
+    def onNext(item: T): Unit
+    def onSubscribe(subscription: Subscription): Unit
```

Comment:
Please order the methods according the "Method Detail" section of the JavaDoc (not by alphabetical order).